firstvoices/pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
-addopts = --ds=firstvoices.settings_test --create-db -nauto
+addopts = --ds=firstvoices.settings_test --reuse-db -nauto
```

Comment:
From my understanding, using `--reuse-db` without `--create-db` prevents it from picking up any new schema changes. Ref: [documentation](https://pytest-django.readthedocs.io/en/latest/database.html#reuse-db-reuse-the-testing-database-between-test-runs). 
On GH actions pipeline, we anyway have to create a new test databse for every run, so not much time saving there. This may also require us to manually clear our local test db after a few runs to clear out any old tables/data. 
Any specific advantages related to celery or this ticket, except for the time gain( on local environment) that I'm missing here ?

Reply:
I reverted this to `reuse-db` since the change was originally made for tests using celery in addition to the time improvement, however I am 100% okay with leaving it as `create-db` if the team would prefer